DogeNet.BLL/Features/Posts/GetPost/GetPostQueryValidator.cs

```diff
@@ -0,0 +1,17 @@
+﻿// <copyright file="GetPostQueryValidator.cs" company="Leobit">
+// Copyright (c) Leobit. All rights reserved.
+// </copyright>
+
+namespace DogeNet.BLL.Features.Posts.GetPost
+{
+    using DogeNet.DAL;
+    using FluentValidation;
+
+    public class GetPostQueryValidator : AbstractValidator<GetPostQuery>
+    {
+        public GetPostQueryValidator(DBContext db)
+        {
+            this.RuleFor(entity => entity.id).NotNull().Must(id => PostChecks.PostIdValid(db, id)).WithMessage("Post not found"); ;
```

Comment:
It would be good to create an extension method for not found messages which will return message based on name of type and some const messages like "Not Found"